dvc/parsing/__init__.py

```diff
@@ -0,0 +1,27 @@
+import logging
```

Comment:
About the general file structure. Maybe let's move `dvc/dvcfile.py` -> `dvc/dvcfile/__init__.py` and `dvc/parsing` -> `dvc/dvcfile/parsing`, since parsing is only for dvcfiles.

Reply:
Yeah, I guess, but I am hesitant to make that decision yet. Everything from this PR is just expendables. I'm sure, it will find a better home.

Reply:
And, hesitant to keep that in dvcfile not yet, as most of the logic might need to be moved to "StageLoader" which in turn has to be moved to dvcfile in the future.

Reply:
@skshetry Ok, your call.